kernelci.org/content/en/docs/org/tsc/_index.md

```diff
@@ -55,14 +55,22 @@ Decisions that always require a vote are:
 * granting or removing any kind of admin rights to individuals
 * making proposals to the LF project board
 
-The result of each vote should be shared on the mailing list and added to the
-[voting records](votes).  It should also include the date when any related
-changes were enacted.  As such they provide a way to keep a formal trace of
-decisions being made.
-
-Votes done during a live meeting require 50% of the TSC members to be voting
-and a majority among these voting members.  Votes done by email or any other
-deferred channel require a majority among all of the TSC members.
+Votes are made on motions with a yes/no choice and have to be proposed by a TSC
+member.  The result of each vote should be shared on the mailing list and added
+to the [voting records](votes).  It should also include the date when any
+related changes are enacted.  As such they provide a way to keep a formal trace
+of decisions being made.
+
+Votes done during a live meeting require a quorum of 50% of the TSC members to
+be voting and a majority among these voting members.  Votes done by email or
+any other deferred channel require a majority among all of the TSC members.
+
+The preferred approach is to vote during TSC meetings.  This should always be
+considered first to encourage discussion and TSC meeting attendance.  If quorum
+is not met during a meeting then the motion can be adjourned to the next
+meeting or brought up for a vote by email.  Votes by email need to be received
+within two weeks.  If no majority has been reached by then, the motion can't be
+approved or "moved" and will be rejected.
```

Comment:
My reading of this means that technically an email vote could be approved if only 1 voter replies with "yes"? (and noone says "no"). Shouldn't an email vote also have to reach a 50% quorum?

Reply:
No for email votes, a 50% majority of the whole TSC needs to be reached.  It's mentioned earlier on the page:

> Votes done during a live meeting require a quorum of 50% of the TSC members to
> be voting and a majority among these voting members.  Votes done by email or
> any other deferred channel require a majority among all of the TSC members.

But thanks for pointing it out, it might be a bit ambiguous here so I'll reword it.

Reply:
Better now, thanks